src/NetworkTest/testQuality/helpers/getPublisherRtcStatsReport.ts

```diff
@@ -0,0 +1,11 @@
+import { OT } from '../../types/opentok';
+
+export default (publisher: OT.Publisher, callback?: (publisherStats?: OT.PublisherRtcStatsReportArr) => void): void => {
+  if (typeof publisher.getRtcStatsReport === 'function') {
```

Comment:
Thanks @jeffswartz. The getRtcStatsReport method is already published from 2.17.6.
We finally decided to go with a non documented method, but it will be documented. I need to update the api-review.